lib/elixir/pages/Guards.md

```diff
@@ -40,8 +41,8 @@ In the example above, we show how guards can be used in function clauses. There
   * function clauses:
 
     ```elixir
-    def foo(term) when is_integer(term), do: term
-    def foo(term) when is_float(term), do: round(term)
+    def type(term) when is_integer(term), do: term
+    def type(term) when is_float(term), do: round(term)
```

Comment:
It should return the atoms `:integer` and `:float` 🤓

Reply:
oops. fixed